assets/translations/en-us.yaml

```diff
@@ -1714,6 +1714,7 @@ configmap:
       protip: Use this area for anything that's UTF-8 text data
     binaryData:
       label: Binary Data
+      noData: No Binary Data available for this ConfigMap...
```

Comment:
There should be a matching message in place for when there's no (non-binary) data. At the moment it shows the keyvalue form but with a placeholder for a row